dxr/plugins/rust/tests/test_data/test_data_types.py

```diff
@@ -0,0 +1,147 @@
+from dxr.testing import DxrInstanceTestCase
+
+class DataTypesTests(DxrInstanceTestCase):
+    def test_struct_name(self):
+        self.found_line_eq('type:NoFields', "struct <b>NoFields</b>;", 6)
```

Comment:
Looks sane to me! If you were using a SingleFileTestCase, you wouldn't even have to specify the line number. (Those are brittle, obviously, and can be a pain.)